php/src/Google/Protobuf/Timestamp.php

```diff
@@ -156,25 +175,5 @@ public function setNanos($var)
         return $this;
     }
 
-    /**
```

Comment:
Please add these methods back.

Reply:
These were removed when I regenerated the protos. Do you know why they were removed?

Reply:
There methods were manually added on generated code. Please add them back.

Reply:
Added them back!